soh/assets/xml/GC_NMQ_D/text/fra_message_data_static.xml

```diff
@@ -0,0 +1,5 @@
+<Root>
```

Comment:
These can probably be combined like `scenes` are

Reply:
Good idea.